openAPI/quran.com/v4.json

```diff
@@ -33,13 +33,13 @@
           "type": "number",
           "minimum": 1,
           "maximum": 114,
-          "description": "The number of the chapter"
+          "description": "The number of the chapter (surah)"
```

Comment:
"Surah" and "Ayah" should be capitalized. This applies to all the changes so will leave just 1 comment for all.

Reply:
Semi-complete. I will triple check my work tomorrow